lib/elixir/src/elixir_env.erl

```diff
@@ -45,13 +45,21 @@ with_vars(Env, #{} = Vars) ->
 reset_vars(Env) ->
   Env#{versioned_vars := #{}}.
 
+set_prematch_from_config(#elixir_ex{} = S) ->
+  Prematch = case elixir_config:get(undefined_variable_as_call) of
+    true -> warn;
+    false -> raise
+  end,
```

Comment:
```suggestion
  Prematch =
    case elixir_config:get(undefined_variable_as_call) of
      true -> warn;
      false -> raise
    end,
```

We don't use anything for formatting Erlang but since the formatter I mostly format it like Elixir code. :D

Reply:
I see, makes sense 🙂

> We don't use anything for formatting Erlang

BTW, is it something that has been considered?